Running tests on Windows

Running npm test gives an error with babel-node / babel-instanbul

I added cross-env to enable setting environment variables in npm script. I want to submit a PR with this change and a fix to get babel working if possible.

> C:\development\parse-server-fork>npm test
> 
> parse-server@2.0.7 pretest C:\development\parse-server-fork
> cross-env MONGODB_VERSION=${MONGODB_VERSION:=3.0.8} ./node_modules/.bin/mongodb-runner start
> 
>   - Starting a MongoDB deployment to test against...
> parse-server@2.0.7 test C:\development\parse-server-fork
> cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node ./node_modules/.bin/babel-istanbul cover -x **/spec/** ./node_modules/.bin/jasmine
> 
> C:\development\parse-server-fork\node_modules\.bin\babel-istanbul:4
> case `uname` in
> ^^^^
> 
> SyntaxError: Unexpected token case
>     at exports.runInThisContext (vm.js:53:16)
>     at Module._compile (module.js:374:25)
>     at Object.Module._extensions..js (module.js:417:10)
>     at Module.load (module.js:344:32)
>     at Function.Module._load (module.js:301:12)
>     at Function.Module.runMain (module.js:442:10)
>     at C:\development\parse-server-fork\node_modules\babel-cli\lib\_babel-node.js:161:27
>     at Object.<anonymous> (C:\development\parse-server-fork\node_modules\babel-cli\lib\_babel-node.js:162:7)
>     at Module._compile (module.js:410:26)
>     at Object.Module._extensions..js (module.js:417:10)
>  npm ERR! Test failed.  See above for more details.

Windows 10 64 bit
PowerShell:
```

> parse-server@2.1.4 pretest B:\Projects\Parse Server\parse-server
> cross-env MONGODB_VERSION=${MONGODB_VERSION:=3.0.8} ./node_modules/.bin/mongodb-runner start


> parse-server@2.1.4 test B:\Projects\Parse Server\parse-server
> cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node $(if [ "$CODE_COVERAGE" = "1" ]; then echo './node_modules/babel-istanbul/lib/cli.js cover -x **/spec/**'; fi;) ./node_modules/jasmine/bin/jasmine.js

./node_modules/jasmine/bin/jasmine.js was unexpected at this time.

B:\Projects\Parse Server\parse-server>  "B:\Projects\Parse Server\parse-server\node_modules\.bin\\node.exe"  "B:\Projects\Parse Server\parse-server\node_modules\.bin\\..\cross-env\bin\cross-env.js" NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node $(if [ "$CODE_COVERAGE" = "1" ]; then echo './node_modules/babel-istanbul/lib/cli.js cover -x **/spec/**'; fi;) ./node_modules/jasmine/bin/jasmine.js
npm ERR! Test failed.  See above for more details.
```
Same output with cmd.

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ node_js:
   - "4.3"
 env:
   global: 
-    - CODE_COVERAGE=1
+    - COVERAGE_OPTION='./node_modules/babel-istanbul/lib/cli.js cover -x **/spec/**'
   matrix:
     - MONGODB_VERSION=2.6.11
     - MONGODB_VERSION=3.0.8

--- a/bin/parse-server
+++ b/bin/parse-server
@@ -1,2 +1,1 @@
-#!/usr/bin/env node
 require("../lib/cli/parse-server");

--- a/package.json
+++ b/package.json
@@ -56,8 +56,9 @@
     "dev": "npm run build && bin/dev",
     "build": "./node_modules/.bin/babel src/ -d lib/",
     "pretest": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=3.0.8} ./node_modules/.bin/mongodb-runner start",
-    "test": "cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node $(if [ \"$CODE_COVERAGE\" = \"1\" ]; then echo './node_modules/babel-istanbul/lib/cli.js cover -x **/spec/**'; fi;) ./node_modules/jasmine/bin/jasmine.js",
+    "test": "cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node $COVERAGE_OPTION ./node_modules/jasmine/bin/jasmine.js",
     "posttest": "mongodb-runner stop",
+    "coverage": "cross-env COVERAGE_OPTION='./node_modules/babel-istanbul/lib/cli.js cover -x **/spec/**' npm test",
     "start": "./bin/parse-server",
     "prepublish": "npm run build"
   },

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "test": "cross-env NODE_ENV=test TESTING=1 ./node_modules/.bin/babel-node $COVERAGE_OPTION ./node_modules/jasmine/bin/jasmine.js",
     "posttest": "mongodb-runner stop",
     "coverage": "cross-env COVERAGE_OPTION='./node_modules/babel-istanbul/lib/cli.js cover -x **/spec/**' npm test",
-    "start": "./bin/parse-server",
+    "start": "node ./bin/parse-server",
     "prepublish": "npm run build"
   },
   "engines": {

